### ISSUE TITLE
Selectively enable quick-lint-js

### ISSUE THREAD

**Issue by ricardobeat** [quick-lint/quick-lint-js#1230]
Is there a way to selectively enable quick-lint-js (using the VS Code plugin)?

Something like requiring a quick-lint-js.config.js file, or *not running* when .eslintrc exists. I work daily with multiple projects and things can get messy with multiple linters running at the same time. This means installing quick-lint-js for a new project is not as easy as it could be in a way that can be shared with teammates.

**Comment by strager** [comment]
> Is there a way to selectively enable quick-lint-js (using the VS Code plugin)?

No, there is currently no way to enable or disable quick-lint-js per-project. I am open to the idea though.

> I work daily with multiple projects and things can get messy with multiple linters running at the same time.

Please help me understand the problem. Is quick-lint-js crashing on some projects? Is it giving diagnostics that it shouldn't give for some projects? Is it giving duplicate diagnostics that ESLint also gives? Or something else?

**Comment by ricardobeat** [comment]
>  Is it giving duplicate diagnostics that ESLint also gives

This. Adding `quick-lint-js` and asking developers to install an editor extension to work on this particular project, means they get duplicate warnings for every project they work on.
